src/core/syscalls/syscall_handler.rs

```diff
@@ -1133,10 +1122,12 @@ mod tests {
 
         // Check StorageReadResponse insert
         assert_eq!(Ok(storage_value), get_big_int(&vm, &relocatable!(2, 2)));
+
+        Ok(())
     }
 
     #[test]
-    fn test_bl_storage_write_hint_ok() {
+    fn test_bl_storage_write_hint_ok() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
same as before. It is ok to unwrap inside tests and because of that there is no need to return a Result in a test.